modules/installation-configuration-parameters.adoc

```diff
@@ -199,7 +199,7 @@ networking:
   - 172.30.0.0/16
   - fd00:172:16::/112
 ----
-endif::[]
+endif::bare,ibm-power,ibm-z,vsphere[]
 
 [NOTE]
 ====
```

Comment:
@juliemathew 

Should I set ifndef clauses for the Network parameters table? As in the IBM Power and IBM Z docs should not include this table?

Reply:
The table should stay in both the IBM Power and IBM Z docs. 
Only the IPv4 restriction needs to removed from P/Z docs.

Reply:
Thanks, @SNiemann15 . The changes look good on this PR?

Reply:
@dfitzmau Yes looks good. 

@juliemathew I also checked the Power docs restriction is removed there too: https://65085--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_power/installation-config-parameters-ibm-power#installation-configuration-parameters-network_installation-config-parameters-ibm-power